packages/shorebird_cli/lib/src/commands/release/release_ios_command.dart

```diff
@@ -193,9 +200,18 @@ ${summary.join('\n')}
       status: ReleaseStatus.active,
     );
 
-    logger
-      ..success('\n✅ Published Release!')
-      ..info('''
+    logger.success('\n✅ Published Release!');
```

Comment:
shouldn't we move the success log until after we've checked the ipaPath? Seems weird to potentially output "success" followed by failure.

Reply:
Fixed.